topology/Harvard University/Harvard University/Harvard FAS Research Computing.yaml

```diff
@@ -0,0 +1,118 @@
+# Production is true if the resource is for production and not testing use
+Production: true
+# SupportCenter is one of the support centers in topology/support-centers.yaml
+SupportCenter: CERN
+
+# GroupDescription is a long description of the resource group; may be multiple lines.
+GroupDescription: This is a set of four frontier-squid proxies for Harvard FAS Research Computing's CVMFS instances
+
+# If you have an up-to-date local git clone, fill GroupID with the output from `bin/next_resource_group_id`
+# Otherwise, leave it blank and we will fill in the appropriate value for you.
+GroupID: 1106
+
+# Resources contains one or more resources in this
+# ResourceGroup. A resource provides one or more services
+Resources:
+  # Resource Name should be a short descriptor of the resource.
+  # e.g. the Center for High Throughput Computing's GlideinWMS Frontend is "CHTC-glidein2"
+  # Resource Names need to be unique across all resources in the OSG.
+  <RESOURCE NAME>:
+    # Active is true if the resource is accepting requests, and false otherwise.
+    # When first registering a resource, set this to false. Set it to true when it's ready for production.
+    Active: true
+    # Description is a long description of the resource; may be multiple lines
+    Description: This is a frontier-squid proxy for Harvard University.
+    # If you have an up-to-date local git clone, fill ID with the output from `bin/next_resource_id`
+    # Otherwise, leave it blank and we will fill in the appropriate value for you.
+    ID: 1123
+    # ContactLists contain information about people to contact regarding this resource.
+    # The "ID" is a hash of their email address available at https://topology.opensciencegrid.org/miscuser/xml
+    # If you cannot find the contact above XML, please register the contact:
+    # https://opensciencegrid.org/docs/common/registration/#registering-contacts
+    ContactLists:
+      # Administrative Contact is one to three people to contact regarding administrative issues
+      Administrative Contact:
+        Primary:
+          Name: Daniel Caunt
+          ID: 25fe4fbd05b36546e5f52f141fb2ef46a50f9cbd
+        # Secondary:
+        #   Name: <FIRSTNAME> <LASTNAME>
+        #   ID: <EMAIL HASH>
+        # Tertiary:
+        #   Name: <FIRSTNAME> <LASTNAME>
+        #   ID: <EMAIL HASH>
+
+      # Security Contact is one to three people to contact regarding security issues
+      Security Contact:
+        Primary:
+          Name: <FIRSTNAME> <LASTNAME>
+          ID: <EMAIL HASH>
```

Comment:
Will this resource have a security contact?

Reply:
```suggestion
      Security Contact: Harvard FAS Research Computing Security Team
        Primary:
          Name: Warren Frame
          ID: <EMAIL HASH>
```